app/models.py

```diff
@@ -1198,6 +1233,14 @@ def get_link(self):
             _external=True,
         )
 
+    @property
+    def template_process_type(self):
+        if self.template_type == SMS_TYPE:
```

Comment:
can we add a doc string explaining why we do this. By default we use process_type in TemplateCategory - unless overridden through the admins for a particular template ( we then use the process_type on the Template table)